lib/miq_environment.rb

```diff
@@ -68,8 +68,9 @@ def self.is_container?
     def self.is_podified?
       return @is_podified unless @is_podified.nil?
 
-      require "container_orchestrator"
-      @is_podified = is_container? && ContainerOrchestrator.available?
+      @is_podified = is_container? &&
+                     File.exist?("/run/secrets/kubernetes.io/serviceaccount/token") &&
+                     File.exist?("/run/secrets/kubernetes.io/serviceaccount/ca.crt")
```

Comment:
Don't like re-declaring these which are constants in ContainerOrchestrator but we can't load that to access them and they are used elsewhere in that class so some duplication is going to be required.

Reply:
Think we should comment that directly 👇 https://github.com/ManageIQ/manageiq/pull/21727#pullrequestreview-886567915